modules/thermal_hydraulics/doc/content/modules/thermal_hydraulics/tutorials/single_phase_flow/step06.md

```diff
@@ -60,27 +44,40 @@ which does not create any of its own Materials:
          block=Closures
          link=False
 
-Then the name we gave this closures object (`no_closures`) is passed to the `closures` parameter.
-Again, this can be done on a global level or per-component basis.
+Then the name we gave this closures object (`no_closures`) is passed to the `closures` parameter in the `hx/pri` component. This will overwrite the `closures` parameter set in the `GlobalParams` block.
+
+
+!listing thermal_hydraulics/tutorials/single_phase_flow/06_custom_closures.i
+         block=hx/pri
+         link=False
 
 ## Materials
 
-In our model, we will use the Churchill's formula for wall friction closure and Dittus-Boelter
-formula for convective wall heat transfer closure
+For our custom closure set, we choose the following expression for the friction factor:
+
+!equation
+f_D = a + b Re^c
+
+where $f_D$ is the Darcy friction factor, $Re$ is the Reynolds number, and $a$, $b$, and $c$ are constant coefficients with $a = 1$, $b = 0.1$, and $c = -0.5$.
 
-The Churchill's formula can be set up by using the [ADWallFrictionChurchillMaterial](ADWallFrictionChurchillMaterial.md)
+ The first step is to define the Reynolds number using an [ADReynoldsNumberMaterial.md] block, then the expression for the friction factor is implemented using a [ADParsedMaterial](ParsedMaterial.md). Note that these materials are defined only on the block where the custom closures are used. They are defined by the other closure set on the other components.
 
 !listing thermal_hydraulics/tutorials/single_phase_flow/06_custom_closures.i
-         block=Materials/f_mat
+         start=Re_mat
+         end=Pr_mat
          link=False
 
-The Dittus-Boelter formula can be set up by using [ADWallHeatTransferCoefficient3EqnDittusBoelterMaterial](ADWallHeatTransferCoefficient3EqnDittusBoelterMaterial.md).
+For the heat transfer, the following expression for Nusselt number is used:
+
+!equation
+Nu = 0.03 Re^{0.9} Pr^{0.5}.
+
+To accomplish this, we define the Prandlt number using an [ADPrandtlNumberMaterial.md] block. The Nusselt number is defined using an [ADParsedMaterial](ParsedMaterial.md), and the heat transfer coefficient is set using an [ADConvectiveHeatTransferCoefficientMaterial.md] block. Again, the block is restricted to the `hx/pri` block to avoid conflicting with the other closure set.
```

Comment:
```suggestion
To accomplish this, we define the Prandtl number using an [ADPrandtlNumberMaterial.md] block. The Nusselt number is defined using an [ADParsedMaterial](ParsedMaterial.md), and the heat transfer coefficient is set using an [ADConvectiveHeatTransferCoefficientMaterial.md] block. Again, the block is restricted to the `hx/pri` block to avoid conflicting with the other closure set.
```